tests/config/drone/banned-password-list.txt

```diff
@@ -2,3 +2,4 @@ password
 12345678
 123
 ownCloud
+ownCloud-1
```

Comment:
Note: this needs to be added because it was added to the web tests by PR https://github.com/owncloud/web/pull/9948
And so when running the web tests in cis, a similar banned-password-list is needed.